### ISSUE TITLE
Container cannot start when `MATOMO_SMTP_AUTH` is defined

### ISSUE THREAD

**Issue by YKWeyer** [bitnami/bitnami-docker-matomo#73]
**Description**

Container cannot start when `MATOMO_SMTP_AUTH` is set

**Steps to reproduce the issue:**

1. Start a container with a valid SMTP configuration, and a valid `MATOMO_SMTP_AUTH` env variable (`Plain`,`Login` or `Crammd5`)

**Describe the results you received:**

Container exits with error:

> The allowed values for MATOMO_SMTP_PROTOCOL are: Plain Login Crammd5

**Describe the results you expected:**

Container should be able to start

**Additional information you deem important (e.g. issue happens only occasionally):**

It looks like a typo in [libmatomo.sh](https://github.com/bitnami/bitnami-docker-matomo/blob/b68a7af8926bf6fdf2145eaa375015abf079665a/4/debian-10/rootfs/opt/bitnami/scripts/libmatomo.sh#L84) :

```shell
! is_empty_value "$MATOMO_SMTP_AUTH" && check_multi_value "MATOMO_SMTP_PROTOCOL" "Plain Login Crammd5"
# Should probably be:
! is_empty_value "$MATOMO_SMTP_AUTH" && check_multi_value "MATOMO_SMTP_AUTH" "Plain Login Crammd5"
```

**Comment by migruiz4** [comment]
Hi @YKWeyer,

Thank you for reporting this issue. I have submitted your fix and a new version including it will be released soon.

Appreciate your collaboration.

**Comment by YKWeyer** [comment]
Thank you, it was fixed for `4.1.0-debian-10-r15` and later :+1:
